examples/quickstart/jupyter-notebooks/druidapi/druidapi/druid.py

```diff
@@ -116,6 +117,33 @@ def datasources(self) -> DatasourceClient:
         if not self.datasource_client:
             self.datasource_client = DatasourceClient(self.rest_client)
         return self.datasource_client
+    
+    def basic_security(self, authenticator, authorizer=None):
+        '''
+        Returns a client to work with a basic authorization authenticator/authorizer pair.
+        This client assumes the typical case of one authenticator and one authorizer. If
+        you have more than one, create multiple clients.
+
+        The basic security API is not proxied through the router: it must work directly with
+        the coordinator. Create an ad-hoc Druid client for your coordinator. Because you have
```

Comment:
```suggestion
        The basic security API is not proxied through the Router: it must work directly with
        the Coordinator. Create an ad hoc Druid client for your Coordinator. Because you have
```